wiki/en/Running-a-Server.md

```diff
@@ -259,7 +222,7 @@ This sets the path to where the Server's recordings will be stored. With this pa
 
 **Note**: When your Server is recording, Clients will display a message that recording is on.
 
-### Directory Server Address
+### Directory Address
```

Comment:
Should still say `Custom Directory address`, as it's the name of the field on the UI and should match exactly.